web-app/less/modules/_landing.less

```diff
@@ -12,17 +21,35 @@
 ul.globus li.fontIcon > a > span.fa,
 ul.globus li.fontIcon > a > span.fal,
 ul.globus li.fontIcon > a > span.far,
-ul.globus li.fontIcon > a > span.fas {color:@mainColor;}
+ul.globus li.fontIcon > a > span.fas {
+  color: @mainColor;
+}
+
 ul.globus li.fontIcon > a > span.fa:hover,
 ul.globus li.fontIcon > a > span.fal:hover,
 ul.globus li.fontIcon > a > span.far:hover,
-ul.globus li.fontIcon > a > span.fas:hover {color:@mainColorHover;}
+ul.globus li.fontIcon > a > span.fas:hover {
+  color: @mainColorHover;
+}
 
 
 .section-body a,
-.section-header a{
+.section-header a {
   color: @secondColor;
 }
-#starredCampaign .starredCampaign-title .starredCampaing-title-text {@title();}
-#starredCampaign .starredCampaign-title .fal{color:@mainColor}
-.participate a:focus {color: @secondColorHover;}
+
+#domain-description-landing-main {
+  background: fade(@mainColor, 20%);
+  color: black;
```

Comment:
- Never use the colors with names. It changes between browsers. 
- The #0000000 (black) is too dark, we are using #545454
- The color of the letter should be in custom.css instead of _landing.less. Is not an info that needs to be preprocessed.